### PR TITLE
docs: describe plugin

### PR DESCRIPTION
**Pull request by tenstad**

### CODE REVIEW

**Review by erikgb — APPROVED** [review]
LGTM

**Comment by anstrom** [comment]
Suggestion:

> If authorized, access is only granted to
the policies of that single role. Even though the JWT satisfies multiple roles'
bound claims. Repeated logins with different role names are thus sometimes
required, resulting in access being split over multiple tokens.

When authenticated, authorization is only performed based on the policies of the autenticated role.

Accessing paths that are governed by different policies may therefore require several logins with separate roles, even though the JWT may satisfy multiple roles' bound claims.

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
...
+**Solution**: The _jwt-auto-roles_ plugin automatically determines which roles'
+bound claims a JWT matches. It grants access to the union of all the matching
+roles' policies. Thus a single login is sufficient to get a token with combined
+access, with the JWT as the sole login parameter.
+
+## Working Details
+
+The plugin currently relies on the builtin _jwt_ plugin for JWT verification and
+policy configuration. It simply determine the roles matching an incoming JWT's
+claims, tries to login to a mount of the builting _jwt_ plugin for each role,
```
Comment (mikaelol) [inline README.md:22]:
```suggestion
claims, tries to login to a mount of the builtin _jwt_ plugin for each role,
```

**Comment by anstrom** [comment]
LGTM

**Comment by mikaelol** [comment]
What about a Usage section?

**Comment by anstrom** [comment]
LGTM

**Review by anstrom — APPROVED** [review]
Good improvements @tenstad!

**Pull request merged**